app/helpers/bootstrap_helper.rb

```diff
@@ -25,6 +25,10 @@ def status_badge(order_detail)
     content_tag :span, order_detail.order_status, class: classes
   end
 
+  def tooltip_hint_icon(text)
+    tooltip_icon "fa fa-question-circle-o", text
+  end
```

Comment:
It looks like this is called in just one place—what do you think about moving the `tooltip_icon` call directly in the view, and avoiding this helper method entirely?

Reply:
That is true: it's only in one place right now. I couldn't find a pattern for hint tooltips like this in the existing app, and figured putting it in a helper establishes it as a standardized pattern for the future. But I could be convinced YAGNI.

Reply:
My vote would be YAGNI, and if we find ourselves repeating this for the third time in the future, we can refactor to a DRY method then 😄I don’t feel too strongly about it, though!